src/js/disability-benefits/526EZ/containers/ConfirmationPage.jsx

```diff
@@ -27,54 +27,79 @@ class ConfirmationPage extends React.Component {
   }
 
   render() {
-    const form = this.props.form;
-    const response = this.props.form.submission.response
-      ? this.props.form.submission.response.attributes
-      : {};
-    const name = form.data.relativeFullName;
-    const benefit = form.data.benefit;
+    const form = /* this.props.form */ initialData;
```

Comment:
We probably want to display the changed data for user testing, so this may not make sense to use `initialData`.

Reply:
Good point, will update.